requirements.txt

```diff
@@ -1,8 +1,8 @@
 requests>=2.32.3,<3
 urllib3>=2.0.5,<3
-aiohttp >=3.10.9
-future
+aiohttp >=3.10.9, <4
+future==1.0.0
 pydantic >=2.0.3, <3
-aenum >= 3.1.11
-python_dateutil >= 2.5.3
-Deprecated
+aenum >= 3.1.11, <4
+python_dateutil >= 2.5.3, <3
+Deprecated==1.2.18
```

Comment:
about `future`, `Deprecated`

If this completely fixes the version, it may cause conflicts with other libraries on the user's side, and the user may not be able to apply security patches. Please only set upper and lower bounds if necessary.